config/environments/test.rb

```diff
@@ -47,4 +47,10 @@
 
   #enables a few aliases - context, should, and should_eventually methods
   config.minitest_spec_rails.mini_shoulda = true
+
+  # High logger level results in less IOps & faster tests
+  config.log_level = :warn
```

Comment:
We also have the test log level set in logging.yaml now, need to work out which one should be in use and if we should change it.